db/migrate/20230220100847_add_devise_to_users.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
### OPTIONAL 

- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉 
- This was added after the previous reviewer requested you remove commented code